### PR TITLE
(CODEMGMT-117) Create Automated Tests for Sprint

### PR DESCRIPTION
**Pull request by carlosvelarde**
2015-03-25

Add two integration tests for User Scenario -
Basic Workflow: Single Environment Specify Module
that is Already Installed, Single Environment
Upgrade Forge Module then Revert Change.

### CODE REVIEW

File: integration/tests/user_scenario/basic_workflow/single_env_module_already_installed.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+test_name 'CODEMGMT-117 - C63601 - Single Environment Specify Module that is Already Installed' 
+
+#Init
+master_certname = on(master, puppet('config', 'print', 'certname')).stdout.rstrip
+environment_path = on(master, puppet('config', 'print', 'environmentpath')).stdout.rstrip
+prod_env_path = File.join(environment_path, 'production')
+git_environments_path = '/root/environments'
+last_commit = git_last_commit(master, git_environments_path)
+prod_env_path = File.join(environment_path, 'production')
+motd_module_init_pp_path = File.join(prod_env_path, 'modules/motd/manifests/init.pp')
```
Comment (cowofevil) [inline integration/tests/user_scenario/basic_workflow/single_env_module_already_installed.rb:13]: Unused variable.

File: integration/tests/user_scenario/basic_workflow/single_env_module_already_installed.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+mod "puppetlabs/motd"
+PUPPETFILE
+
+puppet_file_path = File.join(git_environments_path, 'Puppetfile')
+
+#Manifest
+manifest = <<-MANIFEST
+  class { 'motd':
+    content => '#{motd_contents}',
+}
```
Comment (cowofevil) [inline integration/tests/user_scenario/basic_workflow/single_env_module_already_installed.rb:33]: Brace not indented.

**Comment by cowofevil** [comment]
@carlosvelarde The test looks legit and I think the problem is that the expected outcome when installing an already installed module is not well defined. @andersonmills proposed this test so you should talk to him about what is exactly expected in this scenario.

**Comment by carlosvelarde** [comment]
Thanks, will do.

**Comment by cowofevil** [comment]
@carlosvelarde Is this PR still valid or does it need to be closed?

**Comment by carlosvelarde** [comment]
@cowofevil replaced by 360, closing

**Pull request closed without merging**